src/core.c/CompUnit/Repository/Installation.pm6

```diff
@@ -204,6 +200,53 @@ sub MAIN(:$name, :$auth, :$ver, *@, *%) {
         }
 
         # bin/ scripts
+
+        my $perl-wrapper
+        = do {
+            # code below replaces #ext# with '-j', '-m', etc.
+            #
+            # question here is how to dispatch it: via env or absolute
+            # path to a rakudo executable. this will use env in every
+            # case of *NIX I know of; the bin/rakudo loop is boilerplate.
+            #
+            # lacking both env and raukdo we could try something with
+            # #!/bin/sh -e but that seems to be going too far: give up.
+
+            my $exec = gather {
+                with < /bin/env /usr/bin/env >.first: { .IO.e } -> $env {
+                    take "$env rakudo";
+                } else {
+                    constant SEARCH = 'bin/rakudo';
+                    my $curr    = $.prefix.absolute.IO;
+
+                    note "# Perl-wrapper lacks 'env', using {SEARCH}";
+
+                    loop {
+                        my $raku    = $curr.add( SEARCH );
+
+                        if $raku.f {
+                            take $raku;
+                            last
+                        } elsif $curr eq ( my $next = $curr.parent ) {
+                            die "No '{SEARCH}' above {$.prefix}";
+                        } else {
+                            $curr   = $next;
+                        }
+                    }
+                }
+            };
+
+            # Q: why does this fail with the here-doc indented? 
+
+            ( qq{#!$exec#ext#}, q:to/SHELL/ ).join( "\n" )
```

Comment:
Shelling out to do this is way to fragile. Just remove it.

Reply:
Additionally we don't want to pay the penalty of calculating this every time an installed script is run.